.gitlab-ci.yml

```diff
@@ -51,6 +51,7 @@ test_bulk_update:
   script:
      - python -m unittest tests.test_ingest.TestRun.test_bulk_update
   only:
+    - dcp-552-int-test-download-links
```

Comment:
Are these 2 changes in this config file intentional?

Reply:
no, i will revert them now, did that to test it in gitlab